temporal101/Exercise1.hs

```diff
@@ -3,49 +3,23 @@ module Main where
 import Control.Monad (forever)
 import Control.Monad.IO.Class (liftIO)
 import Control.Monad.Logger (defaultOutput, runStdoutLoggingT)
-import Control.Monad.Trans.Reader (runReaderT)
-import Data.Aeson (FromJSON, ToJSON)
-import Data.Text (Text)
-import Data.UUID qualified as UUID
-import Data.UUID.V4 qualified as UUID.V4
 import DiscoverInstances (discoverInstances)
-import GHC.Generics (Generic)
 import RequireCallStack (RequireCallStack, provideCallStack)
+import SayHello
```

Comment:
might be worth separating this from library imports with a small comment indicating that this is where the workflow definitions come from.